pep.rst

```diff
@@ -435,6 +450,14 @@ In this case the format_spec for the second interpolation is the string
 ``'HTMLNode|str'``; it is up to the ``html`` tag to do something with the
 "format spec" here, if anything.
 
+f-string-style ``=`` Evaluation
+-------------------------------
+
+``mytag'{expr=}'`` is parsed to being the same as ``mytag'expr={expr}``', as
+implemented in the issue `Add = to f-strings for
+easier debugging <https://github.com/python/cpython/issues/80998>`_. We will
```

Comment:
Something more reassuring might be needed here, cause having unrecoverable corner cases might be a deal breaker. Maybe omit the final sentence altogether?

Reply:
Good point, done.